articles/resources/relnotes.md

```diff
@@ -16,6 +16,21 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.xx.xxxx.xxx
```

Comment:
The version number will be in a slightly different format this time to bring it in line with modern NuGet versioning practices (namely, using SemVer 2.0). In particular, the version number will look like `0.12.xxxxxxxx` (no third separator).

Reply:
sounds good, let's update this when we have a version number